### PR TITLE
fix: Add column source to `component_analyses_requests` table.

### PR DESCRIPTION
**Pull request by rafiu007**
This new column will be used to store the source of the request.

### CODE REVIEW

**Review by prashbnair — CHANGES_REQUESTED** [review]
Where is the script to change the data model?

**Comment by rafiu007** [comment]
> Where is the script to change the data model?

I was waiting for +1 for the column name.

File: f8a_worker/storages/postgres.py [hunk 1]
```diff
@@ -231,7 +231,8 @@ def store_api_requests_post(self, arguments):
             ecosystem=arguments['data'].get('ecosystem'),
             user_agent=arguments['data'].get('user_agent'),
             stack_data=json.dumps(arguments['data'].get('packages_list')),
-            manifest_hash=arguments['data'].get('manifest_hash')
+            manifest_hash=arguments['data'].get('manifest_hash'),
+            source=arguments['data'].get('source')
```
Comment (prashbnair) [inline f8a_worker/storages/postgres.py:235]: Are you planning to add the redhat_commons_id later?

Comment (prashbnair) [reply f8a_worker/storages/postgres.py:235]: I would prefer telemetry_id.

Comment (rafiu007) [reply f8a_worker/storages/postgres.py:235]: Yes.first I will modify the api-server then add it here.

**Comment by prashbnair** [comment]
> I was waiting for +1 for the column name.

I would prefer telemetry_id.

**Comment by rafiu007** [comment]
> > I was waiting for +1 for the column name.
> 
> I would prefer telemetry_id.

Okay going ahead with that.

**Review by prashbnair — APPROVED** [review]
Please do not merge until you have everything ready or are willing to push this out to prod in a couple of days.

**Pull request merged**